app/react/browser/createRoutes.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { IndexRoute, Route } from 'react-router';
```

Comment:
It's not :). There are issues with redux actions and transitions are not working.
https://github.com/reactjs/react-router-redux